modules/dnn/src/layers/softmax_layer.cpp

```diff
@@ -208,6 +209,10 @@ class SoftMaxLayerImpl CV_FINAL : public SoftmaxLayer
     }
 #endif
 
+    virtual void finalize(InputArrayOfArrays inputs_arr, OutputArrayOfArrays outputs_arr) CV_OVERRIDE {
```

Comment:
Note that there have been a `finalize` definition at line 124 to 127. Could they be merged into one?

cc @dkurt

Reply:
Indeed, the second one is under `HAVE_OPENCL` directive